src/applications/personalization/profile/components/contact-information/email-addresses/EmailInformationSection.jsx

```diff
@@ -10,61 +10,77 @@ import { signInServiceName as signInServiceNameSelector } from 'platform/user/au
 import { SERVICE_PROVIDERS } from 'platform/user/authentication/constants';
 
 import ProfileInfoTable from '../../ProfileInfoTable';
+import { ProfileInfoCard } from '../../ProfileInfoCard';
+import { Toggler } from '~/platform/utilities/feature-toggles';
 
-const EmailInformationSection = ({ className, signInServiceName }) => {
+const generateRows = signInServiceName => {
   const { link, label: buttonText } =
     SERVICE_PROVIDERS[signInServiceName] || {};
+  return [
+    {
+      title: (
+        <>
+          Contact email
+          <span className="vads-u-color--gray-medium vads-u-display--block vads-u-font-weight--normal">
+            We use this email to send you information.
+          </span>
+        </>
+      ),
+      id: FIELD_IDS[FIELD_NAMES.EMAIL],
+      value: (
+        <ProfileInformationFieldController fieldName={FIELD_NAMES.EMAIL} />
+      ),
+    },
+    {
+      title: 'Sign-in email',
+      value: (
+        <>
+          <span>
+            The email you use to sign in to VA.gov may be different from your
+            contact email.
+          </span>
+          <p>
+            <a href={link} target="_blank" rel="noopener noreferrer">
```

Comment:
good question, does this link need to be tracked on GA? 

* design system for default link guidance mentions [use for navigating to external page](https://design.va.gov/components/link/#when-to-use-a-default-link)
* it doesn't seem like it's currently [being tracked](https://analytics.google.com/analytics/web/?authuser=0#/dashboard/3Z1vT6SlQqevdav1s72QwA/a50123418w177519031p176188361/)

Reply:
This is a good catch. 

I'm not sure why it originally was a bare anchor link, but I think it would make sense to update it to use the va-link instead. I will updated. Thanks!

Reply:
In looking at this, I found that the `va-link` doesn't support the target="_blank" or rel attribute as props that can be passed into it, so going to leave it as it is for now.

It would probably be a good idea to wire up analytics events to it though, so we can track it, which I will add a ticket for in the backlog to explore.